### ISSUE TITLE
additional retrain of tensorflow for new Icon mods

### ISSUE THREAD

**Issue by softttware** [GICodeWarrior/fir#12]
I've been testing this tool out with different icon mods, while mostly work I've found an issue with this relatively newer icon mod. 
this is of no urgency and wanted to know if there's anything to be done because tensorflow is out of my expertise 

https://rainbowbu.itch.io/new-icons-modfoxhole

tysm. <3

**Comment by GICodeWarrior** [comment]
Thanks for sharing.  I'll add this icon mod to the training set.

I aim for 100% accuracy, so if you notice any other issues, please send me some example screenshots.  I will add them to my test suite and attempt to resolve in the next update.  (feel free to DM screenshots via Discord if you prefer not to post)

**Comment by GICodeWarrior** [comment]
@softttware please give it a try now and let me know if you notice any issues.

**Comment by softttware** [comment]
My apologies, I only got to this right now. It still has errors still but I think that is just a problem with resolution and contrast

**Comment by GICodeWarrior** [comment]
Can you DM me an example screenshot?

The tool is designed to work at all common resolutions, etc.

So if it's not working, I'd like to fix it.

**Comment by softttware** [comment]
Here's a screenshot with the item mod. the items that have been causing issues are both Cometa Revolver and .44 Ammo reporting wrongly

![Screenshot War-2024-06-25 22_33_08](https://github.com/GICodeWarrior/fir/assets/119510032/9a1edaa1-d36a-4450-b470-8c168b4581ba)

I don't know how to DM you but if you want to reach out to me on discord i'm @softtware there. 

mod link just in case: https://rainbowbu.itch.io/new-icons-modfoxhole

**Comment by GICodeWarrior** [comment]
Thanks for sharing.

That image seems to work correctly for me.  This is what https://fir.gicode.net/ says:
![2024-06-29T03_26_54 186Z_foxhole-inventory-totals](https://github.com/GICodeWarrior/fir/assets/19803/05aed36f-2d88-4233-ad29-436d70179067)

Please let me know if you see anything wrong there.

If you are running the recognition inside a spreadsheet, that spreadsheet likely needs to be updated.  I put some high level instructions for updating in issue #13.

In the meantime, you could use the "Append to Google Spreadsheet" button to add data to your spreadsheet.

**Comment by softttware** [comment]
I will rebuild FIR and update it and will let you know!

**Comment by softttware** [comment]
reopening this just to ask for the discord in which I can reach out to you, sorry for the trouble

**Comment by GICodeWarrior** [comment]
I believe this issue was resolved.  Feel free to open a new issue if needed.
